cmd/jaas/cmd/bootstrap_unit_test.go

```diff
@@ -18,15 +18,15 @@ import (
 )
 
 type bootstrapCmdSuite struct {
-	client *mocks.MockJIMMClient
+	client *mocks.MockJIMMAPI
```

Comment:
The reason for this change is that the interface was renamed in a previous PR to `JIMMAPI` from `JIMMClient` but the because the interface definition itself wasn't changed the mocks weren't updated. I've updated the interface definition in this PR so I've had to update the mock and usages of it.